pkg/metrics/metric_items.go

```diff
@@ -84,6 +87,11 @@ var MetricsItems = []prometheus.Collector{
 	MigrateGVGCounter,
 	MigrateObjectTimeHistogram,
 	MigrateObjectCounter,
+
+	// golang runtime and process metrics
+	collectors.NewGoCollector(collectors.WithGoCollectorRuntimeMetrics(collectors.GoRuntimeMetricsRule{
+		Matcher: regexp.MustCompile("/.*")})),
+	collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
```

Comment:
Consistent with the above, define a variable.

Reply:
These two metrics are derived from function.